config/stage.json

```diff
@@ -40,7 +40,7 @@
 		"current-site/domain-warning": true,
 		"current-site/notice": true,
 		"current-site/stale-cart-notice": false,
-		"design-picker/goal-centric": true,
+		"desktop-promo": true,
```

Comment:
@p-jackson I think this got added back by accident 😅

Reply:
Ya 😅 I removed it again https://github.com/Automattic/wp-calypso/pull/98622 😁